### ISSUE TITLE
🐧 Norman's Sky ↔ v18.6.2

### ISSUE THREAD

**Issue by Gabriel-p** [itchio/itch-compatibility-watchlist#118]
:rotating_light: game [Norman's Sky](https://nothke.itch.io/normans-sky) is broken for me.

:book: Here's the complete [debug log](https://gist.github.com/87c14c6483157ab0437672d0eccf3225).

:running: elementary OS Freya 64 bit

**Comment by fasterthanlime** [comment]
interesting, Freya doesn't ship with libXcursor1 ?

**Comment by Gabriel-p** [comment]
No idea :(

@danrabbit might be able to share some light into that.

**Comment by danirabbit** [comment]
Wild guess, is this a 32bit game on a 64bit system? Might try installing libxcursor1:i386

**Comment by fasterthanlime** [comment]
@danrabbit correct! I didn't catch that, but the .x86 suffix of the binary (generated by Unity) indicates that it is indeed a 32-bit game. the `uname -a` output indicates that it's a 64-bit system too.

**Comment by Gabriel-p** [comment]
Thanks @danrabbit!

So, did I do something wrong installing or what?

**Comment by fasterthanlime** [comment]
@Gabriel-p you did nothing wrong, it's just that most 32-bit libraries aren't installed by default with 64-bit Linux distributions. Try `sudo apt-get libxcursor1:i386`, that should work fix that problem (there might be other libs missing, you'll get a similar log, you can read it by clicking `Examine..` instead of `Send feedback...`)

**Comment by Gabriel-p** [comment]
Any pointers on where in the log I should look for the missing libraries? It's a huge file, and it's not evident to me where this information is presented.

**Comment by fasterthanlime** [comment]
@Gabriel-p it is indeed pretty big (having a troubleshooting facility right in the itch app and/or falling back to the steamruntime before that is planned), you can simply search for the text `error while loading shared libraries` in your favorite text editor.

**Comment by Gabriel-p** [comment]
Nope, no error with that string. I can see these similar strings along the file:

```
error while launching 38378965-9989-452d-9929-ef383c3c0efd: Error: killed by signal SIGSEGV
error while launching 38378965-9989-452d-9929-ef383c3c0efd: Crash: application crashed. process exited with code 1
error while launching 38378965-9989-452d-9929-ef383c3c0efd: Error: killed by signal SIGSEGV
error while launching 38378965-9989-452d-9929-ef383c3c0efd: Crash: application crashed. process exited with code 1
error while launching 38378965-9989-452d-9929-ef383c3c0efd: Error: killed by signal SIGSEGV
error while launching 38378965-9989-452d-9929-ef383c3c0efd: Crash: application crashed. process exited with code 1
```

**Comment by fasterthanlime** [comment]
Looks like you have all the libraries you need now. However, it looks like the game is segfaulting. I recommend looking at the Unity player log in `~/.config/unity3d/CompanyName/ProductName/Player.log`

**Comment by Gabriel-p** [comment]
Apparently this is the error (from the `Player.log` file):

```
PlayerInitEngineGraphics: GPU not supported; OpenGL 3.2 is required. Your GPU (Mesa DRI Intel(R) Ironlake Mobile ) or OpenGL drivers only supports OpenGL 0.0
```

Not sure how to solve that, but it sound like a handful.

**Comment by fasterthanlime** [comment]
It looks scarier than it is. Unfortunately, OpenGL drivers for Intel devices are pretty poor. You might be able to work around it by passing `-force-opengl` as an argument to the game.

You won't be able to do that via the itch app though, you'll have to open a terminal and run these commands:

``` bash
cd "/home/gabriel/.config/itch/apps/Normans Sky/"
NormansSky_003_Linux/NormansSky.x86 -force-opengl
```

edit: scarier than it looks -> looks scarier than it is 🐧
